exercises/complex-numbers/complex_numbers_test.cpp

```diff
@@ -0,0 +1,372 @@
+#define BOOST_TEST_MAIN
+#include <boost/test/unit_test.hpp>
+#include <complex>
+#include <cmath>
+#include "complex_numbers.h"
+
+// Complex Numbers exercise test case data version 1.0.0
+
+BOOST_AUTO_TEST_CASE(Add_purely_real_numbers)
+{
+    const complex_numbers::Complex c1 {1.0, 0.0};
```

Comment:
I would just add a using definition for the Complex type since you use it soo much.